CMakeLists.txt

```diff
@@ -259,6 +259,12 @@ if(WarpX_PSATD)
     if(WarpX_DIMS STREQUAL RZ)
         target_link_libraries(ablastr PUBLIC blaspp)
         target_link_libraries(ablastr PUBLIC lapackpp)
+
+        # BLAS++ forgets to declare cuBLAS and cuRAND dependencies
+        if(WarpX_COMPUTE STREQUAL CUDA)
+            find_package(CUDAToolkit REQUIRED)
+            target_link_libraries(ablastr PUBLIC CUDA::cudart CUDA::cublas)
```

Comment:
Should we do something similar for HIP/ROC

Reply:
Not currently needed as they do declare dependencies properly on ROCm:
https://bitbucket.org/icl/blaspp/src/2022.05.00/blasppConfig.cmake.in